### PR TITLE
rewrote tagger using ML

### PR DESCRIPTION
**Pull request by oldosamaibraheem11**
Change list:

update SQL-Database with alpha-update

rewrote the tagger to use ML

### CODE REVIEW

File: tagger.py [hunk 1]
```diff
@@ -1,75 +1,33 @@
...
-#tag each word with its POS
-#create new file, one sentence at a time
 
-#Text from 
-#D. Goldhahn, T. Eckart & U. Quasthoff: Building Large Monolingual Dictionaries at the Leipzig Corpora Collection: From 100 to 200 Languages.
-#In: Proceedings of the 8th International Language Resources and Evaluation (LREC'12), 2012
+pathuntagged = input('Enter path to untagged corpus?')
+
+
+FileData = open("/Users/osamaibraheem/Downloads/deu-de_web-wrt_2019_1K/deu-de_web-wrt_2019_1K-sentences.txt", "r")
```
Comment (pnklshh) [inline tagger.py]: Shouldn't we use the value pathuntagged to read the corpus file instead of hardcoded path ?

File: tagger.py [hunk 2]
```diff
@@ -1,75 +1,33 @@
...
+
+nlp = spacy.load("de_dep_news_trf")
```
Comment (pnklshh) [inline tagger.py:9]: Is this file - 'de_dep_news_trf' available in the package ?

File: tagger.py [hunk 3]
```diff
@@ -1,75 +1,33 @@
...
+    newsent = []
+    sentence = lines.split()
+    sentence = sentence[1:]
+    sentence = " ".join(sentence)
+    linessplit = sentence.split()
+    sentence = nlp(sentence)
+    for word in sentence:
+        wordPOS = f'{word.text}/{word.pos_}'
+        newsent.append(wordPOS)
+    return " ".join(newsent)
```
Comment (pnklshh) [inline tagger.py:23]: This is good !

**Review by pnklshh — CHANGES_REQUESTED** [review]
Can you change the path instead of hardcoded path to corpus file ?

File: tagger.py (hunk 2 above)
Comment (oldosamaibraheem11) [reply tagger.py:9]: Yes

**Pull request merged**